css/style.css

```
@@ -537,4 +613,71 @@ background-color: #141B22;
 
 .theme-checkbox:checked + .theme-label .svg-element {
   fill: #1B2226;
-}
\ No newline at end of file
+}
+
+.question-tittle::after {
+  content: url(/img/minus.svg);            /*   I didnt found any other way to solve it, except JS or <input type=checkbox>, but as I know, the 2nd way in this case wont be good, as screen readers may not read it correctly */
+}
+
+details[open] summary::after {
+  content: url(/img/plus.svg);
+}
+
+.joinus {
+  background-color: #0ABE66;
+  background-image: url(/img/star.svg), 
+                    url(/img/star.svg), 
+                    url(/img/star.svg),
+                    linear-gradient(to left, #2333DE 20%, #0ABE66 80%);
+  
+  background-repeat: no-repeat;
+  background-position: -15% 20%, 30% 20%, 110% 80%, left top;
+  background-size: 255px 255px, 137px 137px, 321px 321px,  cover;
+
+  text-align: center;
+  border-radius: 8px;
+}
+
+.subscription-field {
+  background-color: #333;
+	border: none;
```

Code style note: please support the same tabulation space for all properties in a selector
```suggestion
  border: none;
```

Check it everywhere please 🙏🏻

---

You a not the first person to tell me about that, but I dont know how it works, because on my PC its looks like you are asking for

---

Interesting 🤔 I checked again and now it looks outdated. Not sure where I found it 😅 Anyway sorry, my comment looks outdated!